### ISSUE TITLE
RayWenderlich watchOS by Tutorials Updated for Swift 3 and watchOS 3

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-apple-watch-development#8]
This tutorial has been _recently updated:
https://www.raywenderlich.com/145503/watchos-tutorials-updated-swift-3-watchos-3
`Updated by Chris Belanger on October 3, 2016` ... it includes Complications.

+ [ ] investigate if this is "the one" for us!

**Comment by sohilpandya** [comment]
Agree, that this one seems more attractive to me, especially if its updated for watchOS3.

**Comment by sohilpandya** [comment]
I was just talking to someone and they've also recommended RayWenderlich as a good resource. Wondered if it would be possible to subscribe to the site? 

@nelsonic

**Comment by nelsonic** [comment]
@sohilpandya please subscribe and add cost to invoice. 👍
